app/helpers/timeline_helper.rb

```diff
@@ -68,7 +68,7 @@ def public_timeline?
   end
 
   def display_status_class(reservation)
-    reservation.canceled_at ? "canceled" : reservation.order_detail.order_status.to_s.downcase
+    reservation.order_detail.canceled_at ? "canceled" : reservation.order_detail.order_status.to_s.downcase
```

Comment:
Could this method take the order detail as its argument? Would that clean it up a bit?